src/components/image-block/react/ImageBlock.tsx

```diff
@@ -97,13 +99,15 @@ const DEFAULT_PROPS: IDefaultPropsType = {
  * @return The component.
  */
 const ImageBlock: React.FC<ImageBlockProps> = ({
+    actions,
     align = DEFAULT_PROPS.align,
     aspectRatio = DEFAULT_PROPS.aspectRatio,
     className = '',
     captionPosition = DEFAULT_PROPS.captionPosition,
     captionStyle = DEFAULT_PROPS.captionStyle,
     description = DEFAULT_PROPS.description,
     fillHeight = DEFAULT_PROPS.fillHeight,
+    hasActions = DEFAULT_PROPS.hasActions,
```

Comment:
Same as for the avatar

Reply:
Yeah I agree it doesn't have sense here. Was about the commit a change